test/end2end_test.dart

```diff
@@ -102,6 +102,7 @@ void main() {
     }, timeout: const Timeout.factor(2));
   }
 
+  _verifyPackage('dartdoc-0.20.0.json', 'dartdoc', '0.20.0');
```

Comment:
Unrelated?

Reply:
Not unrelated, I'm adding the test because `dartdoc 0.20.0` (and earlier versions) were failing without this change, and the PR unblocks its analysis. If something happens again underneath, we should have better chances to notice it earlier.